clusterloader2/testing/load/modules/reconcile-objects.yaml

```diff
@@ -0,0 +1,299 @@
+## Input params
+# Valid actions: "create", "scale and update", "delete"
+{{$actionName := printf "%s objects" .actionName}}
+{{$namespaces := .namespaces}}
+
+# Derivative variables
+{{$is_creating := (eq .actionName "create")}}
+{{$is_scaling := (eq .actionName "scale and update")}}
+{{$is_deleting := (eq .actionName "delete")}}
+{{$tuning_set := "N/A"}}
+{{if $is_creating}}{{$tuning_set = "RandomizedSaturationTimeLimited"}}{{end}}
```

Comment:
Are you sure? That way we will have three/four separate parameters instead of one. In case you are sure: can we set $actionName conditionally then without passing it as parameter?

Reply:
Why three/four?

I was thinking on a single parameter like "tuningSet: RandomizedSaturationTimeLimited"

Reply:
Ach, sorry. You meant `tuning_set`.

Reply:
Done.